### PR TITLE
Make MulitTable reuse tables

### PR DESCRIPTION
**Pull request by milleruntime**
* This allows restarting rw tests and allow using of the same tables
across multiple threads
* Other various improvements to MultiTable

### CODE REVIEW

File: src/main/java/org/apache/accumulo/testing/randomwalk/multitable/DropTable.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public void visit(State state, RandWalkEnv env, Properties props) throws Excepti
     List<String> tables = (List<String>) state.get("tableList");
 
     // don't drop a table if we only have one table or less
```
Comment (DomGarguilo) [inline src/main/java/org/apache/accumulo/testing/randomwalk/multitable/DropTable.java:36]: This comment might need to be updated to reflect the new  if condition.

Comment (milleruntime) [reply src/main/java/org/apache/accumulo/testing/randomwalk/multitable/DropTable.java]: Fixed in bf9f3fc

**Comment by milleruntime** [comment]
@DomGarguilo think this is OK to merge? I am going to use this for testing.

**Review by DomGarguilo — APPROVED** [review]
Looked through the code and ran rwalk with MultiTable.xml. Everything LGTM

**Pull request merged**